test/relations.test.js

```diff
@@ -3790,7 +3790,7 @@ describe('relations', function() {
           return Person.findById(personId);
         })
         .then(function(person) {
-          person.passportItem().toObject().should.eql({name: 'Jim'});
+          person.passportItem().toObject().should.eql({name: 'Mitsos'});
```

Comment:
>You can also use git blame (or the github UI) to see who wrote this test and ask that person directly.

Hi @bajtos,

It seems this is written by @mitsos1os (community contribution) and got reviewed and landed by you; please see: https://github.com/strongloop/loopback-datasource-juggler/pull/949
and https://github.com/strongloop/loopback-datasource-juggler/commit/bfbdd15f1df00a362df13232a2240a463b7c2263